installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_rules.json

```diff
@@ -6445,6 +6444,27 @@
     "modifiedDate": "2022-07-19",
     "severity": "low",
     "category": "security"
+  },
+  {
+    "ruleId": "trusted_ms_ services_access_storage_account",
+    "ruleUUID": "trusted_ms_ services_access_storage_account",
+    "policyId": "trusted_ms_ services_access_storage_account",
+    "ruleName": "trusted_ms_ services_access_storage_account",
```

Comment:
There is a space in the value of ruleId,ruleuuid,policyId and ruleName:  "trusted_ms_ services_access_storage_account"
Please remove the space in the value